lib/pymedphys/_streamlit/index.py

```diff
@@ -99,7 +118,7 @@ def _get_apps_from_module(module):
 
 
 def main():
-    st.set_page_config(page_title="PyMedPhys", page_icon=FAVICON)
+    st.set_page_config(page_title="PyMedPhys", page_icon=FAVICON, layout="wide")
```

Comment:
If only there was a way to set this for each individual app... It seems you can only do it for the overall project. But overall, I think wide-mode should be better.